### PR TITLE
Implemented POA logic

### PR DESCRIPTION
**Pull request by chinmay-browserstack**
Added POA logic
Related PR https://github.com/percy/cli/pull/1270

### CODE REVIEW

File: Percy/PercyOnAutomate.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+          Log(error.ToString(), "debug");
+      }
+    }
+    public void Screenshot(String name, ScreenshotOptions? options, bool fullScreen) {
+      throw new Exception("Options need to be passed using Dictionary for: " + name);
+    }
+
+    public class Options : Dictionary<string, object> {}
+
+    internal static void Log(String message, String logLevel = "info")
```
Comment (pankaj443) [inline Percy/PercyOnAutomate.cs:55]: can we move common code to IPercy?

File: Percy/lib/IPercyAppiumDriver.cs [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
 using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
+
 namespace PercyIO.Appium
 {
   internal interface IPercyAppiumDriver
   {
     String GetType();
     String Orientation();
+    List<string> getElementIds(List<object> elements);
```
Comment (pankaj443) [inline Percy/lib/IPercyAppiumDriver.cs:11]: we use pascal casing for methods

File: Percy/utils/utils.cs [hunk 3]
```diff
@@ -4,7 +4,7 @@ namespace PercyIO.Appium
 {
   internal class Utils
```
Comment (pankaj443) [inline Percy/utils/utils.cs:5]: rename file to Utils as well

File: Percy/utils/utils.cs [hunk 4]
```diff
@@ -20,5 +20,29 @@ public static Boolean isValidDriverObject(Object obj)
 
       return false;
     }
+
+    public static void Log(String message, String logLevel = "info")
```
Comment (pankaj443) [inline Percy/utils/utils.cs:24]: Sicne you have added Logs to utils please update existing Log calls and remove log code from AppPercy

**Review by ninadbstack — APPROVED** [review]

**Pull request merged**